introduction.md

```diff
@@ -1,2 +1,10 @@
 Hi! I'm Alex Pawlowski, and I am learning how to better use Github. Thanks for the free weekend!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?